app/workers/carry_over_unsubmitted_applications_worker.rb

```diff
@@ -0,0 +1,24 @@
+class CarryOverUnsubmittedApplicationsWorker
+  include Sidekiq::Worker
+
+  def perform
+    unsubmitted_applications_from_earlier_cycle.each do |application_form|
+      CarryOverApplication.new(application_form).call
+    end
+  end
+
+private
+
+  def unsubmitted_applications_from_earlier_cycle
+    # TODO: Remove hard-coded year
+    ApplicationForm
+      .joins(application_choices: :course)
+      .where(submitted_at: nil)
+      .where('courses.recruitment_cycle_year' => 2020)
```

Comment:
```suggestion
      .where('courses.recruitment_cycle_year' => RecruitmentCycle.current_year)
```

Reply:
:+1: this needs to be `RecruitmentCycle.current_year - 1` so that it picks up applications from the previous cycle.